website/models.py

```diff
@@ -289,6 +290,11 @@ class Issue(models.Model):
     cve_id = models.CharField(max_length=16, null=True, blank=True)
     cve_score = models.DecimalField(max_digits=2, decimal_places=1, null=True, blank=True)
     tags = models.ManyToManyField(Tag, blank=True)
+    infringing_domain_name = models.TextField(max_length=100, null=True, blank=True)
```

Comment:
```suggestion
    infringing_domain_name = models.CharField(max_length=100, null=True, blank=True)
```

Reply:
resolved in https://github.com/OWASP-BLT/BLT/pull/2655/commits/3e76e5b033074d878415807aa0742ac40a9e7996